js/apps/admin-ui/src/client-scopes/details/ScopeForm.tsx

```
@@ -156,10 +158,55 @@ export const ScopeForm = ({ clientScope, save }: ScopeFormProps) => {
     convertToFormValues(clientScope ?? {}, setValue);
   }, [clientScope, setValue]);
 
+  const removeEmptyOid4vcAttributes = (
+    values: ClientScopeDefaultOptionalType,
+  ) => {
+    const fieldNames = [
+      "attributes.vc.credential_configuration_id",
+      "attributes.vc.credential_identifier",
+      "attributes.vc.issuer_did",
+      "attributes.vc.expiry_in_seconds",
+      "attributes.vc.credential_build_config.token_jws_type",
+      "attributes.vc.supported_credential_types",
+      "attributes.vc.credential_signing_alg_values_supported",
+      "attributes.vc.verifiable_credential_type",
+      "attributes.vc.credential_build_config.sd_jwt.visible_claims",
+      "attributes.vc.display",
+    ].map((attr) =>
+      convertAttributeNameToForm<ClientScopeDefaultOptionalType>(attr),
+    );
+
+    const cleanedValues = { ...values } as Record<string, unknown>;
+    const cleanedAttributes = {
+      ...(cleanedValues.attributes as Record<string, unknown> | undefined),
+    };
+
+    for (const fieldName of fieldNames) {
+      const attrKey = fieldName.replace(/^attributes\./, "");
+      if (EMPTY_VALUES.includes(cleanedAttributes?.[attrKey] as any)) {
+        delete cleanedAttributes[attrKey];
+      }
+    }
+
+    if (Object.keys(cleanedAttributes).length === 0) {
+      delete cleanedValues.attributes;
+    } else {
+      cleanedValues.attributes = cleanedAttributes;
+    }
+
+    return cleanedValues as unknown as ClientScopeDefaultOptionalType;
+  };
+
+  const onSubmit = (values: ClientScopeDefaultOptionalType) => {
+    const isOid4vc = values.protocol === OID4VC_PROTOCOL;
+    const cleaned = isOid4vc ? removeEmptyOid4vcAttributes(values) : values;
```

Should we log or handle any validation errors here before cleaning? For example, if new OID4VC fields are added later, this logic might silently ignore them. Just asking to confirm the intended behavior.

---

Right , left a comment aswell